finder-app/writer.c

```diff
@@ -0,0 +1,64 @@
+//  /***************************************************************************
+//   * AESD Assignment 2
+//   * Author: Chinmay Shalawadi
+//   * Institution: University of Colorado Boulder
+//   * Mail id: chsh1552@colorado.edu
+//   * References: Stack Overflow, Man Pages
+//   ***************************************************************************/
+
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <fcntl.h>
+#include <syslog.h>
+
+// -----------------------------------main-----------------------------------------------------------
+int main(int argc, char *argv[])
+{
+    // Initializing logging
+    openlog(NULL, 0, LOG_USER);
+
+    // check if there are two arguments
+    if (argc == 3)
+    {
+        // check if string null
+        if (!argv[2])
```

Comment:
@neohere97 think about how you can help the user do the right thing with an error message here.